src/transformers/models/electra/modeling_tf_electra.py

```diff
@@ -1558,9 +1565,13 @@ def __init__(self, config, *inputs, **kwargs):
     @add_start_docstrings_to_model_forward(ELECTRA_INPUTS_DOCSTRING.format("batch_size, sequence_length"))
     @add_code_sample_docstrings(
         processor_class=_TOKENIZER_FOR_DOC,
-        checkpoint=_CHECKPOINT_FOR_DOC,
+        checkpoint="deepset/electra-base-squad2",
```

Comment:
Hi @ydshieh,
Actually, i don't have access for deepset org,
I also tried training it,
![image](https://user-images.githubusercontent.com/26653468/162759853-27658da2-fd02-4cc9-861f-403ed71d119e.png)
It could take more than 16 hours on colab gpu.

Reply:
I updated it with the question-answering electra model that has `tf` checkpoint

Reply:
I was thinking you could create a repo under your account and push the converted the TF checkpoint there.
But it's fine, we can go with the change you have done here.
(unless you have the desire to upload the TF checkpoint under your account)

Reply:
Oops, you don't need to train :) . I will show you how to convert the PT checkpoint to TF version.

Reply:
Ohooh!
I forgot I can always upload someone's checkpoint under my repo! Yea, got your point!
Will upload it!